src/python/settings_default.py

```diff
@@ -0,0 +1,15 @@
+# Copy to settings_secret.py, set strict permissions, and modify.
+# DO NOT PUT THE SECRET FILE IN SOURCE CONTROL!
+
+ENDPOINT = "http://localhost:8080/proctor/Services/exams/expire/"
+CLIENT = "SBAC_PT"
+SSL_CHECKS = True  # Set False for dev servers with invalid SSL. KEEP True FOR PROD!
+
+AUTH_ENDPOINT = "https://your-sso-server.example.com/auth/oauth2/access_token?realm=/your_realm"
```

Comment:
Can you add a comment here explaining that this endpoint and authentication information is related to open AM and not Proctor?  Just to make it really clear